test/e2e/flask/btc/common-btc.ts

```diff
@@ -0,0 +1,59 @@
+import { Mockttp } from 'mockttp';
+import FixtureBuilder from '../../fixture-builder';
+import { withFixtures, unlockWallet } from '../../helpers';
+import { DEFAULT_BTC_ACCOUNT } from '../../constants';
+import { Driver } from '../../webdriver/driver';
+import { createBtcAccount } from '../../accounts/common';
+
+const GENERATE_MOCK_BTC_BALANCE_CALL = (
+  address: string = DEFAULT_BTC_ACCOUNT,
+): { data: { [address: string]: number } } => {
+  return {
+    data: {
+      [address]: 9999,
+    },
+  };
+};
+
+export async function mockBtcBalanceQuote(
+  mockServer: Mockttp,
+  address: string = DEFAULT_BTC_ACCOUNT,
+) {
+  return [
+    await mockServer
+      .forGet(/https:\/\/api\.blockchair\.com\/bitcoin\/addresses\/balances/u)
+      .withQuery({
+        addresses: address,
+      })
+      .thenCallback(() => ({
+        statusCode: 200,
+        json: GENERATE_MOCK_BTC_BALANCE_CALL(address),
+      })),
+  ];
+}
+
+export async function withBtcAccountSnap(
+  {
+    title,
```

Comment:
Nit: We could have use the whole `test` object here instead, so we could have computed the title only once here, rather than having it every calls to `withBtcAccountSnap`